### PR TITLE
POPR-161 Create PreviewLayout view

### PR DESCRIPTION
**Pull request by pamcontin**
# Related Task
- https://app.clickup.com/t/459857/PROP-161

# Proposed Changes
- Fix general styles that where producing bugs
- Create PreviewLayoutContainer reusable in ui
- -Because template preview isn't ready yet, it's prepared for recive it.
preview

- Create PreviewLayout in app

# Amplify Preview
- https://pr-XXXXX.d3gl17ubh0iihv.amplifyapp.com

# Additional Info (Optional)
- Components shown (Contact List) will be move, now it's only to show how components will be position in previewLayout

# How to test this?
- Run app

# Task Checklist
- [x] All new features have been documented
- [x] Conventions and Acceptance criteria have been met
- [x] Code is free of warnings
- [x] ClickUp task has been set to Awaiting Review

### CODE REVIEW

File: frontend/app/tsconfig.json [hunk 1]
```diff
@@ -20,6 +20,6 @@
     "jsx": "react",
     "resolveJsonModule": true
   },
-  "include": ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx"],
+  "include": ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx", "src/layouts/PreviewLayout"],
```
Comment (AgusPk) [inline frontend/app/tsconfig.json]: it's a little odd to add specific folders and files in here, can we look for a more global solution?

Comment (pamcontin) [reply frontend/app/tsconfig.json]: that code generated automaticaly! if you want we can see how to solve it!

File: frontend/app/src/layouts/PreviewLayout/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import { useQuery } from 'react-query';
+import { useTranslation } from 'react-i18next';
+
+type PreviewLayout = {
+  children?: ReactNode;
+};
+
+const PreviewLayout: FC<PreviewLayout> = () => {
+  const { t } = useTranslation(['config', 'translations']);
+  const { data: contacts } = useQuery(['getContacts'], async () => await getContacts('63120fd98028831ecf43b444'));
```
Comment (rosariosm) [inline frontend/app/src/layouts/PreviewLayout/index.tsx:14]: You don't have defined Contact interface. The code should look like `const `{data: Contact[]}`.

Isn't this throwing a typescript error?

**Review by rosariosm — CHANGES_REQUESTED** [review]

File: frontend/app/src/layouts/PreviewLayout/index.tsx (hunk 2 above)
Comment (pamcontin) [reply frontend/app/src/layouts/PreviewLayout/index.tsx:14]: I doesnt throw any errors and it works well like before, all i do was moving that from landing to that component

**Review by rosariosm — APPROVED** [review]

**Pull request merged**